plugins/modules/packaging/os/zypper.py

```diff
@@ -106,6 +106,20 @@
         description:
           - Add additional options to C(zypper) command.
           - Options should be supplied in a single line as if given in the command line.
+    allow_vendor_change:
+        version_added: "2.10"
```

Comment:
Could you remove the `version_added` here and in the next option? Right now this makes the sanity tests fail. (I have a PR where I collect all new version_added's which will be added once it is clear what the correct value is; I regularly update it so this information won't get lost.)

Reply:
Done